nautobot_golden_config/nornir_plays/config_intended.py

```diff
@@ -127,11 +130,14 @@ def config_intended(job_result, log_level, job_class_instance, qs, device_to_set
 
         logger.debug("Run nornir render config tasks.")
         # Run the Nornir Tasks
-        nr_with_processors.run(
+        results = nr_with_processors.run(
             task=run_template,
             name="RENDER CONFIG",
             logger=logger,
-            device_to_settings_map=device_to_settings_map,
-            job_class_instance=job_class_instance,
+            device_to_settings_map=job.device_to_settings_map,
+            job_class_instance=job,
             jinja_env=jinja_env,
         )
+
+    if results.failed:
```

Comment:
This is what it was 2 weeks ago, https://github.com/nautobot/nautobot-app-golden-config/commit/9963daebead33548616bcd974d48bccf4214a041#diff-ab5cdfc66117f42882540610a6b1e7962bd8062ab2922d843a5e88bffce4293bL141-L144 

I am not tracking what this is doing. So far it seems like if there is a results.failed, it would raise an empty message that would obscure what the actual issue is. 

@jdrew82 @jeffkala and I should sync to understand what was wrong with the initial solution, what this should do, and what the path forward should be with the shared context of all of our learning and understanding.

Reply:
Per our discussion I've re-added the try/except blocks to catch the NornirNautobotExceptions being thrown from generalized Exceptions being thrown should now be logged and possibly re-raised if the Exception is found to be from nornir-nautobot or nautobot-app-nornir. This should ensure that Exceptions like "missing network_driver" or ValueError/IntegrityErrors being thrown should be caught and raised.